tests/v2/waku_relay/test_wakunode_relay.nim

```diff
@@ -67,11 +65,11 @@ procSuite "WakuNode - Relay":
 
   asyncTest "Messages are correctly relayed":
     let
-      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      nodeKey1 = generateKey()
```

Comment:
In general `nodeKeyxx` is never used so perhaps.

`node1 = WakuNode.new(newEcdsaKey(), ValidIpAddress.init("0.0.0.0"), Port(60410))`

see also other tests.
 just a suggestion though.

Reply:
I did a find/replace. What you suggest is a ton of work (note how many times these methods are used). 

I suggest that every time one touches a test suite changes this issue 🙂

Reply:
sounds good thanks, resolving